tests/client/coll.rs

```diff
@@ -14,7 +12,7 @@ fn find_and_insert() {
 
     // Insert document
     let doc = doc! {
-        "title" => Bson::String("Jaws".to_owned())
+        "title" => (Bson::String("Jaws".to_owned()))
```

Comment:
`$` would probably be sufficient actually. I'm just concerned that parenthesis would add a lot of confusion because that overrides how parenthesis work in Rust. If we could get something like `"title" => $Bson::String("Jaws".to_owned())` that would actually be a good alternative too.